rules/docker-fpm-frr.mk

```diff
@@ -28,7 +28,7 @@ SONIC_DOCKER_IMAGES += $(DOCKER_FPM_FRR)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_FPM_FRR_DBG)
 
 $(DOCKER_FPM_FRR)_CONTAINER_NAME = bgp
-$(DOCKER_FPM_FRR)_RUN_OPT += --privileged -t
+$(DOCKER_FPM_FRR)_RUN_OPT += -t --cap-add=NET_ADMIN --cap-add=SYS_ADMIN
```

Comment:
Why is sys_admin required here?

Reply:
zebra will unexpectedly exited due to missing the required capabilities.

```
May  2 22:36:14.048947 str-s6000-acs-10 INFO bgp#supervisord: zebra privs_init: initial cap_set_proc failed: Operation not permitted
May  2 22:36:14.049851 str-s6000-acs-10 INFO bgp#supervisord: zebra Wanted caps: cap_net_admin,cap_net_raw,cap_sys_admin=p
May  2 22:36:14.050687 str-s6000-acs-10 INFO bgp#supervisord: zebra Have   caps: cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_net_admin,cap_net_raw,cap_sys_chroot,cap_mknod,cap_audit_write,cap_setfcap=p
May  2 22:36:14.064956 str-s6000-acs-10 INFO bgp#supervisord 2023-05-02 22:36:14,061 INFO exited: zebra (exit status 1; not expected)
```